dirty_cat/test/test_gap_encoder.py

```diff
@@ -159,36 +165,35 @@ def test_score(n_samples=70):
     score_X2 = enc.score(X2)
     # Check that two identical columns give the same score
     assert score_X1 * 2 == score_X2
-    return
 
 
 @pytest.mark.parametrize("missing", ['zero_impute', 'error', 'aaa'])
-def test_missing_values(missing):
+def test_missing_values(missing: str) -> None:
     observations = [['alice', 'bob'], ['bob', 'alice'], ['bob', np.nan],
                     ['alice', 'charlie'], [np.nan, 'alice']]
     observations = np.array(observations, dtype=object)
     enc = GapEncoder(handle_missing=missing, n_components=3)
     if missing == 'error':
-        with pytest.raises(ValueError, match=r'Input data contains missing values.'):
+        with pytest.raises(ValueError,
+                           match='Input data contains missing values'):
             enc.fit_transform(observations)
     elif missing == 'zero_impute':
         enc.fit_transform(observations)
         enc.partial_fit(observations)
     else:
-        with pytest.raises(ValueError, match=r"handle_missing should be either "
-                                             r"'error' or 'zero_impute', got 'aaa'"):
+        with pytest.raises(ValueError,
+                           match=r"handle_missing should be either "
+                                 r"'error' or 'zero_impute', got 'aaa'"):
             enc.fit_transform(observations)
 
 
-def profile_encoder(Encoder, init):
-    # not an unit test
-
-    from dirty_cat.datasets import fetch_employee_salaries
+def profile_encoder(init: str) -> None:
```

Comment:
I wonder if we should not remove this code: if I understand well, it is dead code that is never run.

Let's consider this in a separate issue / PR, as I want to merge this

Reply:
Added to the scope of #298